packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -32,7 +33,7 @@ const ContractTypeWidget = observer(
             active_symbols: { active_symbols },
             ui: { is_mobile },
         } = useStore();
-        const { symbol } = useTraderStore();
+        const { symbol, show_description, setShowDescription } = useTraderStore();
```

Comment:
do we need a store for show_description and setshowdescription? Feels like it can be done with local storage. similar to other modals

Reply:
This flag is needed for opening description if user clicks on 'Learn more'. So, I need to listen to its changes (when it becomes `true`) inside of `useEffect` in another component. I thought, that if some value is changing and I need to know about it, it's better to keep it in store, not in localStorage.